# Salmon-tximport-deseq2

# Data
S. cerevisiae WT vs snf2 KO mutant RNA-seq data with 7 technical and 48 biological replicates (336 total) of each condition

# Description 	
High-throughput RNA sequencing (RNA-seq) is now the standard method to determine differential gene expression. Here, a 48 replicate, two condition RNA-seq experiment was designed specifically to test assumptions about RNA-seq read count variability models and the performance of methods for differential gene expression analysis by RNA-seq. Samples were run on an Illumina HiSeq for 50 cycles single-end and included ERCC RNA spike-ins. The high-replicate data allowed for strict quality control and screening of 'bad' replicates. The experiment allowed the effect of bad replicates to be assessed as well as providing guidelines for the number of replicates required for differential gene expression analysis and the most appropriate statistical tools. The mapping between technical replicates and biological replicates is provided via FigShare http://dx.doi.org/10.6084/m9.figshare.1416210The gene read counts are also available on FigShare: https://dx.doi.org/10.6084/m9.figshare.1425503 https://dx.doi.org/10.6084/m9.figshare.1425502
